README.md

```
@@ -149,6 +149,18 @@ check binary under $GOPATH/bin or $GOBIN
 If you want to update binaries, the following command.
            $ gup update mimixbox
 ```
+
+### Machine-readable JSON output (for scripting / CI)
+`list`, `check`, and `update` accept `--json`, printing a JSON array instead of the human-readable output (which stays the default).
+
+```shell
+$ gup check --json | jq -r '.[] | select(.status == "update-available") | .name'
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)